lib/ansible_tower_client/hash_model.rb

```diff
@@ -71,6 +71,7 @@ def convert_value(key, value, parent)
 
       def add_accessor_methods(method, key)
         return if modelized_list.include?(method.to_s)
+        key = key.to_s
```

Comment:
Why a symbol is required here? I am not convinced we ever need to fix `hash_model`.